e2e-tests/src/test/java/uk/nhs/adaptors/gp2gp/e2e/EhrExtractTest.java

```diff
@@ -423,7 +422,17 @@ private static String getEnvVar(String name, String defaultValue) {
         return value;
     }
 
-    private String buildAbsentAttachmentFileName(String conversationId, String documentId) {
-        return "AbsentAttachment" + conversationId + "/" + documentId + ".txt";
+    private boolean nameHas(Object name, String startingValue, String endingValue){
```

Comment:
Could we maybe rename this to something that implies that we are checking the start and end of a value? Or even just separate the two assertions for starts/ends with on line 406?